.bazelrc

```diff
@@ -0,0 +1,4 @@
+# This will be the default in bazel.
+# https://blog.bazel.build/2018/01/19/config-parsing-order.html
+startup --expand_configs_in_place
```

Comment:
The link (`https://blog.bazel.build/2018/01/19/config-parsing-order.html`) says it should no longer be configurable at `0.13`, which is the latest release. Which bazel version are you running?

Reply:
Looks like I built it from head shortly pre-0.11. I'll update, thanks!

Reply:
Sure thing ;)

Reply:
Removed.